h2o-r/h2o-package/R/predict.R

```diff
@@ -0,0 +1,49 @@
+#'
+#' H2O Prediction from R
+#'
+#' Provides the method h2o.predict with which you can predict a MOJO or POJO Jar model
+#' from R.
+#'
+
+
+
+#' @param model  String with file name of MOJO or POJO Jar
+#' @param json  JSON String with inputs to model
+#' @param classpath  (Optional) Extra items for the class path of where to look for Java classes, e.g., h2o-genmodel.jar
+#' @return Returns an object with the prediction result
+#' @importFrom jsonlite fromJSON
+#' @examples
+#' \donttest{
+#' library(h2o)
+#' h2o.predict_json('~/GBM_model_python_1473313897851_6.zip', '{"C7":1}')
+#' h2o.predict_json('~/GBM_model_python_1473313897851_6.zip', '{"C7":1}', c(".", "lib"))
+#' }
+#' @usage
+#' h2o.predict_json(model, json, classpath)
+#' @name h2o.predict_json
+
```

Comment:
As commented above, better to keep documentation comments (`#' ...`) in single chunk, then there is no need to repeat `@name h2o.predict_json`.
Additionally you are putting multiple variables into h2o package namespace, they will act as a global variables for any function in h2o namespace. None of those seems to be time consuming, I would simply put them inside the function. Unless you need to use them in another function?